common/rfb/VNCServerST.h

```diff
@@ -116,6 +116,7 @@ namespace rfb {
     const core::Point& getCursorPos() const { return cursorPos; }
     const char* getName() const { return name.c_str(); }
     unsigned getLEDState() const { return ledState; }
+    bool getDesktopStartupInProgress() const { return desktopStartupInProgress; }
```

Comment:
This seems very implementation focused. How about `isDesktopReady()`, to keep with the other naming?